src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForLoopStatement.cs

```diff
@@ -1247,24 +1247,24 @@ public void Next()
 string expectedOperationTree = @"
 IForLoopStatement (LoopKind.For) (OperationKind.LoopStatement) (Syntax: 'for (d.Init ... }')
   Condition: IUnaryOperatorExpression (UnaryOperationKind.DynamicTrue) (OperationKind.UnaryOperatorExpression, Type: System.Boolean) (Syntax: 'd.Done')
-      Operand: IOperation:  (OperationKind.None) (Syntax: 'd.Done')
```

Comment:
@jinujoseph This seems to be caused by two PRs merged around the same time w/o running tests on up-to-date code?